packages/microsoft_kiota_azure/lib/src/access_token.dart

```diff
@@ -0,0 +1,11 @@
+part of '../microsoft_kiota_azure.dart';
+
+/// Represents the access token response from any credential provider.
+/// The [token] is the actual access token, the [expiresOn] is the expiration time of the token, the [refreshAt] is the time when the token should be refreshed and the [tokenType] is the type of the token.
```

Comment:
```suggestion
/// The [token] is the actual access token.
/// The [expiresOn] is the expiration time of the token.
/// The [refreshAt] is the time when the token should be refreshed.
/// The [tokenType] is the type of the token.
```
nit for consistent documentation style.